README.md

```diff
@@ -36,3 +36,5 @@ Waiting on dependencies
 
 ### Sass styling setup
 - [ ] Initial research and setup complete, fixes pending before merging to master
+
```

Comment:
> (need to talk to Mads about the yaml file data). 

Is this still something you want to talk about? I think you could also just go with arbitrary numbers (or just copy-paste the numbers you need from either the yaml file or other sources), instead of wrestling with that massive file. At least for the first versions, then you can challenge yourself later on if needed 😄